cmd/builder/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/knadh/koanf/parsers/yaml v0.1.0
 	github.com/knadh/koanf/providers/env v1.0.0
 	github.com/knadh/koanf/providers/file v1.1.2
-	github.com/knadh/koanf/providers/fs v0.1.0
+	github.com/knadh/koanf/providers/fs v1.0.0
```

Comment:
This is a major version change, needs to update the code

Reply:
the import isn't changed, should be ok as is